packages/rules/src/bannedDependencies.ts

```diff
@@ -13,7 +13,7 @@ import path from "path";
 import * as r from "runtypes";
 import { PackageDependencyGraphService } from "./util/packageDependencyGraphService";
 
-const Options = r.Union(
+const BannedDependenciesOptions = r.Union(
```

Comment:
i think the rest of the repo names this as `Options` though

Reply:
I thought the type has to match the const declaration

Reply:
i don't quite follow, why does the name matter?

Reply:
cuz I need to export the type for tests. figured exported types should have unique names.
i can remove tho, since it works so long as only these options are imported in only this rules test suite